doc/Media2.xml

```diff
@@ -2167,6 +2167,244 @@
         </variablelist>
       </section>
     </section>
+	<section xml:id="section_k4m_bcd_yyz">
+      <title>Audio Clip Management</title>
+      <section>
+        <title>Audio Clip Configuration</title>
+        <para>This section is about uploading one or more pre-recorded audio clips to the device.
+          The uploaded audio clips can be played by the user on need basis or on configured schedule time or 
+		  when the associated event is triggered. Audio Clip configuration has the following parameters:</para>
+        <itemizedlist>
+          <listitem>
+            <para>Enabled [xs:boolean] – Enables/disables audio clip configuration.</para>
+          </listitem>
+          <listitem>
+            <para>Name [xs:string] – Name of the audio clip configuration.</para>
+          </listitem>
+          <listitem>
+            <para>Token [tt:ReferenceToken] – Unique Token associated with the audio clip configuration.</para>
+          </listitem>
+          <listitem>
+            <para>Type [xs:string] - Format of the audio clip. For definitions see tt:SupportedAudioClipFormat parameter.</para>
+          </listitem>          
+          <listitem>
+            <para>RepeatCycles [xs:int] - Number of times the audio clip can be repeated when it is played. The value 0 means no repeat
```

Comment:
Suggestion: often number of plays is defined as '1' meaning single while 0 means endless loop.

Reply:
currently 0  means no repeat i.e play the audio clip, does not repeat after the first play. 1 means play the audio clip and repeat once after the first play. -1 means repeat the audio clip infinite after first play until the clip is stopped by the user operation.

we have ONVIF wush ticket  for this parameter  usage - https://wush.net/trac/onvif/ticket/2816

if we change 0 to endless loop, if user does not want to repeat  we need to provide the value.